getting-started/setup-building.rst

```diff
@@ -455,7 +455,7 @@ Install dependencies
 ====================
 
 This section explains how to install additional extensions (e.g. ``zlib``)
-on :ref:`Linux <deps-on-linux>` and :ref:`macOS`.
+on Linux and macOS.
```

Comment:
Are these links no longer needed due to the introduction of OS-specific tabs?

Reply:
Yes, the targets are just above this header, so there's no point linking to the current location.